train.py

```diff
@@ -1,152 +1,192 @@
-'''
+"""
 Created on May 6, 2018
 @author: jana
-'''
+
+Module purpose description
+"""
+# Python standard library
+import random
+import sys
+
+# 3rd party modules
+import numpy as np
 import torch
 import torch.autograd as autograd
 import torch.nn as nn
 import torch.optim as optim
-from data_processing import load_data, prepareMolData, prepareMinibatches, context_dictionary, loadProteinRestrictions
+
+# Our own
+from data_processing import (
+    load_data,
+    prepareMolData,
+    prepareMinibatches,
+    context_dictionary,
+    loadProteinRestrictions,
+)
```

Comment:
Why do we break this line into several ones? I don't think it improves readability.